src/server/src/test/java/io/cassandrareaper/acceptance/ReaperAuthIT.java

```diff
@@ -1,57 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.cassandrareaper.acceptance;
```

Comment:
why is this file removed?

Reply:
it shouldn't have been added in the first place, I'm just cleaning up as I'm working on integration test classes (I know you don't like mixing changes though ;) )

Reply:
it should be removed in a separate PR then. (or just ninja fix it in master.)